FEM/Output.cpp

```diff
@@ -561,6 +567,24 @@ ios::pos_type COutput::Read(std::ifstream& in_str,
             _tecplot_zones_for_mg = true;
             continue;
         }
+        // for tecplot: Number output files for multiple output on same domain
+        if (line_string.find("$NUMBER_TEC_OUTPUT") != string::npos)
+        {
+            _number_outputs = true;
+            continue;
+        }
+        // For tecplot node-nr output JM
+        if (line_string.find("$NODE_INDEX_OUTPUT") != string::npos)
+        {
+            _node_output = true;
```

Comment:
the variable name might be misleading. how about renaming it to "_node_id_output"?